drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -227,12 +227,21 @@ function DragAndDropTemplates(configuration) {
     };
 
     var feedbackTemplate = function(ctx) {
-        var feedback_display = ctx.feedback_html ? 'block' : 'none';
         var properties = { attributes: { 'aria-live': 'polite' } };
+        var messages = ctx.overall_feedback_messages || [];
+        var feedback_display = messages.length > 0 ? 'block' : 'none';
+        var feedback_messages = messages.map(function(message) {
+            var selector = "p.message";
+            if (message.message_class) {
+                selector += "."+message.message_class;
+            }
+            return h(selector, {innerHTML: message.message}, []);
```

Comment:
Note also that some of these message are not expected to have HTML-- for example, the messages in FeedbackMessages. It is worth spending some time thinking about what can be escaped; this is an area where we've been notoriously sloppy and open to vulnerabilities. And since this code is in a separate repo, it doesn't benefit from the safe linting checks in edx-platform.